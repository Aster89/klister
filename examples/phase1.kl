#lang kernel

[import [shift kernel 1]]


[meta
  [define m-impl
    [lambda [s]
      (syntax-case s
        [[vec [_ e]] [pure e]])]]]

[define-macros ([m m-impl])]

[example [m 1]]
