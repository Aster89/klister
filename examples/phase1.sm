#lang kernel
[meta [import kernel lambda]]

[meta
  [define m-impl
    [lambda [s]
      (syntax-case s
        [[vec [_ e]] [pure e]])]]]

[define-macros ([m m-impl])]

[example [m 1]]
