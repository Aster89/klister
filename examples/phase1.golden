Signal 1 : Signal
